service/header/core_listener.go

```diff
@@ -79,6 +79,11 @@ func (cl *CoreListener) listen(ctx context.Context, sub <-chan *types.Block) {
 
 			eh, err := MakeExtendedHeader(ctx, b, comm, vals, cl.dag)
 			if err != nil {
+				if err == errEmptyBlock {
+					log.Debugw("core-listener: ", err)
```

Comment:
We should probably log more details about the block here (height, time received, etc.)